src/epub/text/chapter-20.xhtml

```diff
@@ -18,7 +18,7 @@
 			<p>… Quickly, metallically, distinctly, do the thoughts rap in the head. An invisible aero carries me into the blue height of my beloved abstractions. And I see how there in the height, in the purest rarified air, my judgment about the only “right” bursts with a crack, like a pneumatic tire. I see clearly that only an atavism, the absurd superstition of the ancients, gives me this idea of “right.”</p>
-			<p>There are ideas of clay and ideas moulded of gold, or of our precious glass. In order to know the material of which an idea is made, one needs only to let fall upon it a drop of strong acid. One of these acids was known to the ancients under the name of <i xml:lang="la">reductio ad absurdum</i>. This was the name of it, I think. But they were afraid of this poison; they preferred to believe that they saw <em>heaven</em>, even though it was a toy made of clay, rather than confess to themselves that it was only a blue nothing. We on the other hand (Glory to the Well-Doer!), we are adults and we have no need of toys. Now if we put a drop of acid on the idea of “right”.⁠ ⁠… Even the ancients (the most mature of them) knew that the source of right was⁠—might! Right is a function of might. Here we have our scale: on the one side an ounce, on the other a ton. On one side “I,” on the other “we,” the United State. Is it not clear? To assume that I may have any “right” as far as the State is concerned, is like assuming that an ounce may equilibrate a ton in a scale! Hence the natural distribution: tons⁠—rights, grams⁠—duties. And the natural road from nothingness to greatness, is to forget that one is a gram and to feel that one is one-millionth of a ton!</p>
+			<p>There are ideas of clay and ideas moulded of gold, or of our precious glass. In order to know the material of which an idea is made, one needs only to let fall upon it a drop of strong acid. One of these acids was known to the ancients under the name of <i xml:lang="la">reductio ad absurdum</i>. This was the name of it, I think. But they were afraid of this poison; they preferred to believe that they saw <em>heaven</em>, even though it was a toy made of clay, rather than confess to themselves that it was only a blue nothing. We on the other hand (Glory to the Well-Doer!), we are adults and we have no need of toys. Now if we put a drop of acid on the idea of “right.”⁠ ⁠… Even the ancients (the most mature of them) knew that the source of right was⁠—might! Right is a function of might. Here we have our scale: on the one side an ounce, on the other a ton. On one side “I,” on the other “we,” the United State. Is it not clear? To assume that I may have any “right” as far as the State is concerned, is like assuming that an ounce may equilibrate a ton in a scale! Hence the natural distribution: tons⁠—rights, grams⁠—duties. And the natural road from nothingness to greatness, is to forget that one is a gram and to feel that one is one-millionth of a ton!</p>
```

Comment:
I'd leave the italics here because it clarifies that we're *calling* something that.

Reply:
Yep, I reverted this one.